plugin.audio.100fm/changelog.txt

```diff
@@ -0,0 +1,17 @@
+v0.2.0
+- Official 100FM Radius radio station add-on
```

Comment:
That means that in opposite to the previous version of this plugin, this time the station decided to adopt this plugin and offer this add-on to their listeners as additional platform